meson.build

```diff
@@ -571,16 +571,38 @@ if build_standalone
   subdir('po')
   subdir('libfwupdplugin')
   subdir('contrib')
-
-  # the plugins each add the gresource files into an array, and then in src:
-  # * for -Dgresource_quirks=enabled, they get built into ./src/fwupd and the files are ignored
-  # * for -Dgresource_quirks=disabled, the files get installed into quirks.d
   subdir('plugins')
   subdir('src')
-
 endif
 subdir('docs')
 
+# install the quirk files
+if get_option('compressed_quirks')
+
+  # append all the quirks into one big file
+  builtin_quirk = custom_target('builtin-quirk',
+    input: plugin_quirks,
+    output: 'builtin.quirk',
+    capture: true,
+    command: ['cat', '@INPUT@'],
+  )
+
+  # gzip it
+  gzip = find_program('gzip')
```

Comment:
Did you profile lzma vs gzip?

Reply:
lzma was smaller, but needs libxmlb 0.3.3 -- and we only BR on 0.1.13 for RHEL and Ubuntu LTS.

Reply:
Okay, maybe a future optimization some day we can bump that. Leave a comment for future selves the numbers in difference to justify looking into it someday

Reply:
Fixed.